website/src/app/[lang]/[region]/(website)/partners/(components)/PartnerHome.tsx

```diff
@@ -0,0 +1,253 @@
+		orgLongName: currentNgo!['org-long-name'],
+		partnershipStart: currentNgo!['partnership-start'],
+		orgDescriptionParagraphs: currentNgo!['org-description-paragraphs'],
+		quote: currentNgo!['org-quote'] ?? null,
+		quoteAuthor: currentNgo!['org-quote-author'] ?? null,
+		quotePhoto: currentNgo!['org-quote-photo'] ? image_base_path.concat(currentNgo!['org-quote-photo']) : null,
+		orgFoundation: currentNgo!['org-foundation'],
+		orgHeadquarter: currentNgo!['org-headquarter'],
+		orgWebsite: currentNgo!['org-website'] ?? null,
+		orgFacebook: currentNgo!['org-facebook'] ?? null,
+		orgInstagram: currentNgo!['org-instagram'] ?? null,
+		orgLinkedIn: currentNgo!['org-linkedin'] ?? null,
+		orgYoutube: currentNgo!['org-youtube'] ?? null,
+		orgFundRaiserText: currentNgo!['org-fundraiser-text'] ?? null,
+		orgSlug: currentNgo!['org-slug'],
+	};
+	const showVisitOnline: boolean = !!(
+		ngoHoverCard.orgInstagram ||
+		ngoHoverCard.orgFacebook ||
+		ngoHoverCard.orgWebsite ||
+		ngoHoverCard.orgLinkedIn ||
+		ngoHoverCard.orgYoutube
+	);
+
+	const showFundRaiser: boolean = !!ngoHoverCard.orgFundRaiserText;
+	return (
+		<div className="flex items-center justify-center">
+			<div className="sm:w-3/4">
+				<div className="relative">
+					<Image
+						className="h-auto w-full rounded-t-lg"
+						src={ngoHoverCard.orgImage}
+						width="48"
+						height="48"
+						alt="Organization Photo"
+						unoptimized
+					/>
+					<div className="absolute bottom-0 left-0 h-32 w-full bg-gradient-to-t from-black to-transparent">
+						<div className="text-accent absolute bottom-0 left-0 px-8 py-4">
+							<Typography size="5xl" weight="medium">
+								{ngoHoverCard.orgLongName}
+							</Typography>
+						</div>
+					</div>
+				</div>
+				<div className="px-8 pb-10 pt-2">
+					<div className="flex flex-col gap-2 p-0 pb-8 pt-2 sm:flex-row sm:items-center sm:justify-between">
+						<div className="pb-4 text-center sm:order-2 sm:flex-shrink-0 sm:pb-0 sm:text-right">
+							<Typography size="md" weight="normal">
+								{translations.partnerSince} {ngoHoverCard.partnershipStart}
+							</Typography>
+						</div>
+						<div className="flex flex-wrap items-center justify-center gap-2 sm:justify-start">
+							<RecipientsBadge
+								{...recipientsBadge}
+								isInsideHoverCard={true}
+								translatorBadgeRecipients={translations.badgeRecipient}
+								translatorBadgeRecipientsBy={translations.badgeRecipientBy}
+								translatorBadgeActive={translations.badgeActive}
+								translatorBadgeFormer={translations.badgeFormer}
+								translatorBadgeSuspended={translations.badgeSuspended}
+							/>
+							<Badge className="bg-primary hover:bg-primary text-primary space-x-2 bg-opacity-10 px-4 py-2 hover:bg-opacity-100 hover:text-white">
+								{countryBadge?.countryFlagComponent || <SL className="h-5 w-5 rounded-full" />}
+								<Typography size="md" weight="normal" className="text-inherit">
+									{currentNgo!['org-long-name']}
```

Comment:
Should show _country_ not _org name_
<img width="563" alt="Screenshot 2024-12-05 at 11 43 33" src="https://github.com/user-attachments/assets/f7de2866-5c8e-4912-beeb-13a824021b1d">

Reply:
Resolved, was a strange mistake from my end